### ISSUE TITLE
Feature: cli option to skip generating postman .json file

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#792]
Also, add session ID to the file path

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [2.0.14](https://github.com/open-wa/wa-automate-nodejs/compare/2.0.13...2.0.14) (2020-09-08)

- Release 2.0.14 [`166e974`](https://github.com/open-wa/wa-automate-nodejs/commit/166e9748111845cfba661b17cb2a97706a6cfa93)
- Fix: Implemented #792 [`d2f8587`](https://github.com/open-wa/wa-automate-nodejs/commit/d2f85878e958c19a99e676e4c77a1c95c737d3ec)
- Feature: Implemented `--api-host` on cli #793 [`e78f147`](https://github.com/open-wa/wa-automate-nodejs/commit/e78f1476ea34ec0fa3ff30d81ded7154fe4fdc8e)
- Fix: #791 [`0181c8f`](https://github.com/open-wa/wa-automate-nodejs/commit/0181c8f1a06a6cd859b1606e6397d99859a5db79)
- Fix: #775 [`e5f67b9`](https://github.com/open-wa/wa-automate-nodejs/commit/e5f67b92014ec59d5ab6fb119240146dfe2cc4ab)
